content/en/mobile/_index.md

```diff
@@ -99,6 +99,97 @@ You can also specify a dashboard that opens by default when you tap on an SLOs w
 
 **Note**: If you do not specify a dashboard that opens by default, tapping an SLOs widget opens the Datadog app.
 
+#### Create an SLOs widget
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+
+- Long press on your home screen.
+- Tap the “+” button on the top left corner of the screen.
+- Search for "Datadog" widgets.
+- Select your prefered size (small shows one SLO and medium shows one SLO along with a visualized timeframe of its health).
+- Drag the widget to your desired, on-screen location.
+
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+- Long press on your home screen.
+- Tap the "Widgets" button on your home screen editor. If you have app shortcuts, it might appear as just an icon on the top right corner of the bubble.
+- Drag the widget to your desired, on-screen home screen.
+- Resize the widget to fit your preference; it will always shows one SLO. If you size the widget to take up the width of your mobile home screen, it will display the selected SLO along with a visualized timeframe of its health.
+
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Edit an SLOs widget
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+
+- Long press on the widget to configure.
+- Tap on "Edit Widget".
+- Tap on the "Choose" button next to the "SLO" label to choose an SLO to track.
+- Depending on the SLO chosen, a "Timeframe" label may appear; tap on the "Choose" button next to the "Timeframe" label to choose the SLO timeframe. 
+- Tap on the "Choose" button next to the "Dashboard to open" label to choose a dashboard that opens when the SLOs widget is tapped. 
+- Tap out of the widget to validate your selection and exit the configuration screen.
+
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+- Tap on the widget title to configure.
+- Tap on the "Selected SLO" section to choose an SLO to track.
+- Tap on the "Selected Time Window" section to choose the SLO timeframe.
+- Tap on the "Dashboard to open" section to choose a dashboard that opens when the SLOs widget is tapped
+- Tap the "Save" or "Apply" button to validate your selection and exit the configuration screen.
```

Comment:
```suggestion
- Tap the widget title to configure.
- Tap "Selected SLO" to choose an SLO to track.
- Tap "Selected Time Window" to choose the SLO timeframe.
- Tap "Dashboard to open" to choose a dashboard that opens when the SLOs widget is tapped.
- Tap "Save" or "Apply" to validate your selection and exit the configuration screen.
```
Does it really say "Selected Time Window"? or should it be "Selected Time Frame"?